src/web/web_middleware.ml

```diff
@@ -17,13 +17,12 @@ module Security = Web_middleware_security
 module Static = Web_middleware_static
 module Session = Web_middleware_session
 module Authn = Web_middleware_authn
+module MakeCsrf = Web_middleware_csrf.Make
```

Comment:
@aronerben Please make the naming consistent to other middlewares. (Look at session middleware)